### ISSUE TITLE
jwdid 

### ISSUE THREAD

**Issue by sahileco** [friosavila/stpackages#17]
I am trying for post estimation after jwdid but it shows varlist required such as for estate event even though i run the example given in stata document.

**Comment by friosavila** [comment]
Can you update jwdid

**Comment by sahileco** [comment]
Hi, thank you for your reply. I am using this - 
which jwdid
C:\Users\LENOVO\ado\plus\jwdid.ado
*!v2.11 Bug with trtvar

**Comment by friosavila** [comment]
ok so two other things to try
1. Restart your stata session (assuming you have not done so)
2. update stata itself
update all

then try,
jwdid, example

and see if it works.
Lastly, if it doesn't, create a logfile

log using test, text replace
set tracedepth 3
set trace on
jwdid, example
set trace off
log close

F

**Comment by mcaceresb** [comment]
@friosavila I also ran into this issue on Stata 15 and 16 but not 18. Log: [jwdid_test.log](https://github.com/user-attachments/files/19046696/jwdid_test.log)

**Comment by friosavila** [comment]
Thank you very much. I was using and testing this with Stata 18 19 and 15. Didnt realize that using Stata 17 would create an issue
I suspect its an unintended error in Stata itself that was never patched. 
@mcaceresb I will be making a correction in the spirit of what you suggest

**Comment by Lu-Yunzhi** [comment]
You are right, and that's why I call it "brute-forced". To me (or my current project), weights are not that important. It's a somewhat "hot-fix" to make the program run. I think @mcaceresb proposed a more decent way to deal with this bug.

**Comment by friosavila** [comment]
Yes it’s fixed
I ahould contact Stata about it, but at this point I doubt anything could
be done

**Comment by mcaceresb** [comment]
@friosavila I think there's some other parsing issues with weights in margins but that's not really something you can fix; at least this bypasses the main problem for this particular case. Thanks!
